RockLib.Logging.Extensions/RockLibLoggerProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace RockLib.Logging.Extensions
+{
+    /// <summary>
+    /// Provides the logger for RockLib and Microsoft.Extensions.Logging
+    /// </summary>
+    public class RockLibLoggerProvider : Microsoft.Extensions.Logging.ILoggerProvider
+    {
+        private readonly string _rockLibName;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Logger"/> class.
+        /// </summary>
+        public RockLibLoggerProvider()
+            : this(null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Logger"/> class.
+        /// </summary>
+        /// <param name="rockLibName">The name of the logger.</param>
+        public RockLibLoggerProvider(string rockLibName)
```

Comment:
Should probably rename the `rockLibName` parameter (and corresponding field) to `rockLibLoggerName` to match the extension method parameter.